### PR TITLE
Link to main page after successful registration

### PR DESCRIPTION
**Pull request by Zash**
Someone who registers via the web might also be interested in proceeding to the web portal. Perhaps it's just me as a developer and/or admin.

Suggestions for proper wording, or better solutions would be appreciated.

### CODE REVIEW

File: snikket_web/templates/invite_success.html [hunk 1]
```diff
@@ -16,5 +16,6 @@ <h1>{% trans site_name=config["SITE_NAME"] %}Successfully registered on {{ site_
 	{%- endcall -%}
 	<p>{% trans %}You can now set up your legacy XMPP client with the above address and the password you chose during registration.{% endtrans %}</p>
 	<p>{% trans %}You can now safely close this page.{% endtrans %}</p>
+	<p><a href="/">{% trans %}Check out the web portal something something?{% endtrans %}</a></p>
```
Comment (horazont) [inline snikket_web/templates/invite_success.html:19]: Use `href="{{ url_for('main.login') }}"` to go to the login page directly (and without relying on magic paths :))

I'm also not quite happy with the wording ;). 

How about integrating this with the previous sentence:

`You can now safely close this page or proceed to the <a href="...">web login</a>.`

Though that might cause people to believe that they can chat from there…

Comment (mwild1) [reply snikket_web/templates/invite_success.html:19]: This is related to #95 - this is indeed a weird dead-end page for users currently. A link to the login would be an improvement indeed.

How about:

> You may close this page, or log in to the web portal to **manage your account**.

Comment (horazont) [reply snikket_web/templates/invite_success.html:19]: :+1:

Comment (Zash) [reply snikket_web/templates/invite_success.html:19]: Thanks, will incorporate feedback and drop the draft status if there's nothing else.

**Comment by Zash** [comment]
@horazont pointed me to #16 which would help here as well.

File: snikket_web/templates/invite_success.html [hunk 2]
```diff
@@ -15,6 +15,7 @@ <h1>{% trans site_name=config["SITE_NAME"] %}Successfully registered on {{ site_
 		{% trans %}Copy address{% endtrans %}
 	{%- endcall -%}
 	<p>{% trans %}You can now set up your legacy XMPP client with the above address and the password you chose during registration.{% endtrans %}</p>
-	<p>{% trans %}You can now safely close this page.{% endtrans %}</p>
+	<p>{% trans login_url=url_for('main.login') %}You can now safely close this page, or log in to the web portal to <a href="{{ login_url }}">manage your account</a>{% endtrans %}</p>
```
Comment (horazont) [inline snikket_web/templates/invite_success.html:18]: Missing trailing period.

**Review by horazont — CHANGES_REQUESTED** [review]

File: snikket_web/templates/invite_success.html (hunk 2 above)
Comment (Zash) [reply snikket_web/templates/invite_success.html]: Fixed.

**Review by horazont — APPROVED** [review]

**Pull request merged**